Source/WebCore/platform/graphics/mac/controls/SwitchMacUtilities.h

```diff
@@ -29,7 +29,7 @@
 namespace WebCore::SwitchMacUtilities {
 
 IntSize cellSize(NSControlSize);
-FloatSize visualCellSize(NSControlSize, const ControlStyle&);
+FloatSize visualCellSize(IntSize, const ControlStyle&);
```

Comment:
Can you explain the motivation behind the type change? It seems like we simply moved the call to `cellSize` out of the method, but using the same `NSControlSize` and `ControlStyle`?

Reply:
It did not seem sound for visualCellSize() to call cellSize() directly as that would bypass the class method. And also, for the thumb we want to invoke cellSize() before visualCellSize() as it's easier to create the thumb variant from the logical size.